crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -558,6 +559,17 @@ en:
       active: 'Active'
       active_repo_tooltip: 'Managed nodes will be automatically configured to use repositories only if they are marked as active'
 
+  ceph_pre_upgrade:
+    index:
+      title: 'Preparation of Ceph Nodes'
+      hint: |
+        Some explanatory text why this is needed.
+        This might take more lines.
```

Comment:
How about something like this?

> The upgrade from SUSE Enterprise Storage 2.1 to 4 must be performed as outlined in the [SUSE Enterprise Storage 4 documentation](docserv.suse.de/documents/Storage_4/ses4-admin/html/cha.ceph.upgrade.html). For this to work, the Ceph nodes must first be put into a prepared state, so that Crowbar will leave them alone for the duration of the upgrade.